packages/frontend-2/pages/developer-settings.vue

```diff
@@ -220,18 +280,27 @@ import {
   BookOpenIcon,
   TrashIcon,
   PencilIcon,
-  LockOpenIcon
+  LockOpenIcon,
+  XMarkIcon
 } from '@heroicons/vue/24/outline'
 import type {
   TokenItem,
-  ApplicationItem
+  ApplicationItem,
+  AuthorizedAppItem
 } from '~~/lib/developer-settings/helpers/types'
 import {
   developerSettingsAccessTokensQuery,
-  developerSettingsApplicationsQuery
+  developerSettingsApplicationsQuery,
+  developerSettingsAuthorizedAppsQuery
 } from '~~/lib/developer-settings/graphql/queries'
 import { useQuery } from '@vue/apollo-composable'
 
+// TODO: Hide first party apps like in FE1
```

Comment:
Just checking this is something for future, and not something you meant to come back to